src/uwtools/drivers/forecast.py

```diff
@@ -153,9 +153,11 @@ def resources(self, platform: dict) -> Mapping:
             "walltime": "00:01:00",
         }
 
-    def run(self) -> None:
+    def run(self) -> bool:
         """
-        Runs FV3 either as a subprocess or by submitting a batch script.
+        Runs FV3 either locally or via a batch-script submission.
+
+        :return: Did the FV3 run exit with success status?
```

Comment:
Oh, it sure does just move merrily along when the executable fails.

Reply:
The exit status of NWP codes has been historically unreliable (Fortran for a long time didn't support Unix-style exit codes, but I believe it does now, and many codes now use it), but we might as well give them the benefit of the doubt and try to use them.